data/gegno/power.txt

```diff
@@ -0,0 +1,119 @@
+outfit "R01 Skirmish Battery"
+	category Power
+	licenses
+		"Gegno Civilian"
+	plural "R01 Skirmish Batteries"
+	cost 17500
+	mass 12
+	thumbnail outfit/gbatterytiny
+	"energy capacity" 1121
+	"outfit space" -12
+	description "Skirmish batteries are very common amongst ships across Gegno space. They can be either a perfectly convenient storage of battery power to offset the load of larger Gegno generators, or a nice addition to add on top of them where more energy would be needed."
+
+outfit "R02 Battlezone Battery"
+	category Power
+	licenses
+		"Gegno Civilian"
+	plural "R02 Battlezone Batteries"
+	cost 32000
+	mass 23
+	thumbnail outfit/gbatterysmall
+	"energy capacity" 2237
+	"outfit space" -23
+	description "Battlezone batteries are primarily used on the Gegno's medium-range combat vessels, holding a decent amount of power for long lasting engagements. They also fit nicely on larger vessels as an extra power source to feed their larger systems."
+
+outfit "R03 Warforge Battery"
+	category Power
+	licenses
+		"Gegno Civilian"
+	plural "R03 Warforge Batteries"
+	cost 61700
+	mass 41
+	thumbnail outfit/gbatterymed
+	"energy capacity" 4469
+	"outfit space" -41
+	description "Warforge batteries have proved their usefulness over the years as large, sturdy battery packs. They support the equally large volleys that are fired from Gegno ships, and help maintain an excess of power for engine usage."
+
+outfit "R04 Crusade Battery"
+	category Power
+	licenses
+		"Gegno Civilian"
+		"Vi Evocati"
+		"Vi Centurion"
+		"Vi Lord"
+	plural "R04 Crusade Batteries"
+	cost 136000
+	mass 83
+	thumbnail outfit/gbatterylarge
+	"energy capacity" 8885
+	"outfit space" -83
+	description "As the largest battery mounted on Gegno ships, the Crusade Battery supplies almost enough power as a small generator, enough to support the massive warships of both the Scin and the Vi. However, they fall under the capacity of similarly-sized human batteries due to inefficient handling of their scale."
```

Comment:
The plurals were a typo- I've given the rest plurals and fixed the identifications. 

Decided to change the Brawl Core to be Cell, since cell is more unique than generator. Also changed the description to reflect it. 

Changed "enough" to "as much". Going to try to think of ways to solve the "don't produce power" comparison.

Once I get a chance, I'm going to go through all files and reorganize them to be the same as the base game. The reason I did not do this beforehand was because the editor I was using (quyukk's) rearranges things out of order again after use.

Reply:
The "inefficient handling of their scale" is to imply their inner workings are not engineered properly for a battery their size. Not sure how to word that better (I'm sure there are many ways I can't think of right now), but I will give it thought.

Reply:
`However, the Gegno are not able to increase density with size as effectively as is done with human batteries, so these fall behind similarly sized options there.`
Possibly a bit long.
`However, these fall behind similarly sized human batteries; the Gegno have not been able to increase density as they scale up the size as effectively.`
I don't know if this is correct use of a semi-colon.

Reply:
"However, the Gegno are inefficient at increasing density with size, as done with human batteries, so these fall behind simiarly sized options."